packages/dapp-svelte-wallet/react-ui/src/components/AppBar.js

```diff
@@ -30,16 +34,62 @@ const useStyles = makeStyles(theme => ({
   appBarSection: {
     display: 'flex',
     flexDirection: 'row',
+    alignItems: 'center',
     padding: '4px',
+    margin: '8px',
     height: '100%',
   },
+  navMenuButton: {
+    display: 'none',
+    [theme.breakpoints.down('sm')]: {
+      display: 'block',
+    },
+  },
 }));
 
 const AppBar = () => {
+  const theme = useTheme();
   const classes = useStyles(useTheme());
+  const [drawerOpened, setDrawerOpened] = useState(false);
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+
+  const toggleDrawer = open => event => {
+    if (
+      event.type === 'keydown' &&
+      (event.key === 'Tab' || event.key === 'Shift')
+    ) {
+      return;
+    }
+
+    setDrawerOpened(open);
+  };
+
+  useEffect(() => {
+    if (!isMobile) setDrawerOpened(false);
+  }, [isMobile]);
+
+  const navMenuButton = isMobile ? (
+    <div className={`navMenuButton ${classes.navMenuButton}`}>
```

Comment:
Can you please break this out into a NavMenuButton component?

Reply:
Oh, and for these things, `const navMenuButton = isMobile && (....);` is better style.  When any falsy thing is substituted in a component with `{navMenuButton}`, it won't render.

Reply:
Done & done, and added tests